modules/op-pipelines-as-code-command-reference.adoc

```diff
@@ -115,3 +115,121 @@ The `-f` option can also accept a directory path and apply the `tkn pac resolve`
 You can override the default information gathered from the Git repository by specifying parameter values using the `-p` option. For example, you can use a Git branch as a revision and a different repository name.
 
 |===
+
+cel::
+
+:FeatureName: The `tkn pac cel` command
+include::snippets/technology-preview.adoc[]
+
+[options="header"]
+|===
+
+| Command | Description
+
+| `tkn pac cel` | Evaluate CEL (Common Expression Language) expressions against webhook payloads and headers. This command is intended for use by admins to test and debug event filtering logic in {pac} using payload and header data obtained directly from the Git provider webhook delivery interface.
```

Comment:
i think we need to mentions somehow that it is intended to be run by an admin since it require to have access to the payload of the event and the header to properly work